src/main/java/com/dev/hibernate/model/Author.java

```diff
@@ -0,0 +1,41 @@
+package com.dev.hibernate.model;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "author")
+public class Author {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String name;
+    private String surname;
```

Comment:
краще так, але це не критично
```suggestion
    private String lastname;
```